mods/ra/rules/aircraft.yaml

```diff
@@ -255,6 +255,10 @@ HELI:
 		Description: Helicopter gunship armed\nwith multi-purpose missiles.\n  Strong vs Tanks, Aircraft\n  Weak vs Infantry
 	Health:
 		HP: 120
+	RejectsOrders:
+		Except: Attack
```

Comment:
This isn't working for me.
Perhaps I am missing something.

I am able to give non-Attack orders to my longbow just fine (as long as it is airborne).